CFE-500: Updates spec annotation in nodeobservability.yaml file (#49585)
openshift/openshift-docs: modules/node-observability-create-custom-resource.adoc

[IMPORTANT]
====

> The worker nodes which match the `nodeSelector` specified in `NodeObservability` CR will be rebooted. It might take 10 or more minutes to complete.

Reviewer: ```suggestion
The worker nodes matching the `nodeSelector` specified in `NodeObservability` CR are rebooted. It might take 10 or more minutes to complete.
```

Author: fixed.

====

> When you apply the `NodeObservability` CR, it creates the necessary machine config and machine config pool CRs to enable the CRI-O profiling on the worker nodes which match the `nodeSelector`.

Reviewer: ```suggestion
When you apply the `NodeObservability` CR, the necessary machine config and machine config pool CRs are created to enable the CRI-O profiling on the worker nodes matching the `nodeSelector`.
```

Author: fixed.

[IMPORTANT]
====

> The worker nodes which are only specified in `NodeObservability` CR are rebooted. It might take 10 or more minutes to complete.

Reviewer 2: ```suggestion
The nodes which match the nodeSelector specified in `NodeObservability` CR will be rebooted. It might take 10 or more minutes to complete.
```

====

> When you apply the `NodeObservability` CR, it creates the necessary machine config and machine config pool CRs to enable the CRI-O profiling on the worker nodes.

Reviewer 2: ```suggestion
When you apply the `NodeObservability` CR, it creates the necessary machine config and machine config pool CRs to enable the CRI-O profiling on the nodes which match the nodeSelector.
```

Author: fixed
